### PR TITLE
feat: added navbar

### PR DESCRIPTION
**Pull request by SirSimon162**
### Your checklist for this pull request

🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [ ] Make sure you are making a pull request against the **default branch** (left side). Also you should start _your branch_ off _default branch_.
- [ ] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] Check your code additions will fail neither code linting checks nor unit test.
- [ ] I have added necessary documentation (if appropriate)

### Description

Please describe your pull request.
Added new navbar

❤️Thank you!

### Post merge checklist

- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Comment by SirSimon162** [comment]
@Shurtu-gal please review

File: src/components/marginals/Navbar/styles.jsx [hunk 1]
```diff
@@ -12,68 +12,62 @@ const fadeDown = keyframes`
...
     }
+    &::before {
+      content: '';
+      position: absolute;
+      width: 0;
+      height: 1.5px;
+      bottom: 0;
+      left: 50%;
+      display: block;
+      background: linear-gradient(88.29deg, #ffb8ff 1.56%, #b9cdff 94.54%);
```
Comment (Shurtu-gal) [inline src/components/marginals/Navbar/styles.jsx]: Use the variable declared in global CSS.
`
background: var(--brand-gradient);
`

File: src/components/marginals/Navbar/DesktopNav.jsx [hunk 2]
```diff
@@ -60,7 +61,7 @@ function DesktopNav() {
             </ul>
           </NavCenter>
           <NavRight>
-            <AuthButton outline />
+            <AuthButton />
```
Comment (Shurtu-gal) [inline src/components/marginals/Navbar/DesktopNav.jsx]: Delete the authButton and use https://github.com/dscnitrourkela/project-raisin/blob/main/src/components/shared/Button.jsx with an outline prop.

**Review by Shurtu-gal — CHANGES_REQUESTED** [review]

**Review by Shurtu-gal — APPROVED** [review]
LGTM 🚀

**Pull request merged**